### PR TITLE
Typo fix

### PR DESCRIPTION
**Pull request by Bowrna**
I have added a typo fix in VS Code `Extensions` view part in TESTING.rst file under the title `To run unit tests from the Visual Studio Code:`

---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/main/UPDATING.md).

### CODE REVIEW

File: CONTRIBUTORS_QUICK_START.rst [hunk 1]
```diff
@@ -147,29 +147,36 @@ Pyenv and setting up virtual-env
 
 3. Add the lines suggested at the end of installation  to ~/.bashrc
 
-4. Restart your shell so the path changes take effect and verifying installation
+4. If you are in macOS Big Sur 11.4, pyenv may face issues. It can be fixed by adding the below command to ~/.zshrc
```
Comment (uranusjr) [inline CONTRIBUTORS_QUICK_START.rst]: This combined with the previous point (which mentioned `~/.bashrc`) is very confusing. Does this only apply if you are using Zsh? What about Bash? Doesn't pyenv emit "lines suggested at the end of the installation" when run in Zsh?

Comment (Bowrna) [reply CONTRIBUTORS_QUICK_START.rst]: No pyenv doesn't emit the suggested lines at the end of the installation. Let me confirm if it applies to bash too.

Comment (Bowrna) [reply CONTRIBUTORS_QUICK_START.rst]: I have followed the instructions in [this](https://github.com/pyenv/pyenv/issues/1948)  issue link to add these docs. It worked for me when I added the below instruction in the ~/.zshrc file.

I checked for bash and found [this](https://github.com/pyenv/pyenv#basic-github-checkout) guide where they have given similar instructions for bash too. 

maybe I have to improve the instructions or point to the pyenv README.md for correct installation instruction as they have updated for zsh and bash? Please share your comments @uranusjr

Comment (Bowrna) [reply CONTRIBUTORS_QUICK_START.rst]: It was not updated in the Pyenv readme when I raised and fixed this issue. But now the [README](https://github.com/pyenv/pyenv#readme) for Pyenv is updated with installation for specific OS and different shell terminals too. 

To verify what instructions pyenv is emitting during installation I tried to install again and got the below message. 

Screenshot 2021-10-27 at 6 44 59 PM

can I modify the instruction to point to README for pyenv link to follow the instructions for proper installation? @uranusjr

Comment (uranusjr) [reply CONTRIBUTORS_QUICK_START.rst]: Yeah, that sounds like a good idea. Modify step 3 to say something like "Configure your shell's environment for Pyenv as suggested on [link to Pyenv's readme]"

Comment (potiuk) [reply CONTRIBUTORS_QUICK_START.rst]: Yep. Indeed this will be much better.

**Review by uranusjr — APPROVED** [review]

**Review by potiuk — APPROVED** [review]

**Pull request merged**